lib/aerospike/value/value.rb

```diff
@@ -22,14 +22,11 @@
 module Aerospike
   # Polymorphic value classes used to efficiently serialize objects into the wire protocol.
   class Value #:nodoc:
-    INTEGER_RANGE = Range.new(-2**63, 2**63 - 1).freeze
 
     def self.of(value)
       case value
-      when nil
-        res = NULL
       when Integer
-        if INTEGER_RANGE.cover?(value)
+         if value.bit_length < 64
```

Comment:
this is 2x faster than `cover?`, makes a difference when serializing a lot of integers:

```ruby
require 'benchmark/ips'

INTEGER_RANGE = Range.new(-2**63, 2**63 - 1).freeze

GC.disable


in_range = 32492349
out_of_range = 2**67

Benchmark.ips do |x|
  x.report('cover? in range') { INTEGER_RANGE.cover?(in_range) }
  x.report('bit_length in range') { in_range.bit_length < 64 }
  x.report('cover? out of range') { INTEGER_RANGE.cover?(out_of_range) }
  x.report('bit_length out of range') { out_of_range.bit_length < 64 }

  x.compare!
end
```

```
Warming up --------------------------------------
     cover? in range   231.972k i/100ms
 bit_length in range   330.636k i/100ms
 cover? out of range   158.398k i/100ms
bit_length out of range
                       282.780k i/100ms
Calculating -------------------------------------
     cover? in range      6.656M (±11.0%) i/s -     32.940M in   5.013587s
 bit_length in range     18.404M (± 3.5%) i/s -     91.917M in   5.000726s
 cover? out of range      8.072M (± 3.5%) i/s -     40.391M in   5.009805s
bit_length out of range
                         16.168M (± 7.5%) i/s -     80.310M in   5.000645s

Comparison:
 bit_length in range: 18404208.3 i/s
bit_length out of range: 16168037.6 i/s - 1.14x  slower
 cover? out of range:  8072316.9 i/s - 2.28x  slower
     cover? in range:  6655993.4 i/s - 2.77x  slower
```